thesis/front/abstract.tex

```diff
@@ -32,43 +32,41 @@
     With the advance of science and technology,
 the computing power of smart devices has increased,
 and the size getting more compact,
-these make people easier to record sounds.
-However, as the recording equipment becomes easier to carry with,
-it also makes eavesdropping becomes more difficult to prevent.
+these make peoples easier to record sounds.
```

Comment:
I don't think we should use "peoples" in this sentence

Reply:
any advising subjects?

Reply:
just use "people" will be enough

Reply:
or we can just use "user" instead?

Reply:
`people`